script/travis/install.sh

```diff
@@ -40,7 +40,7 @@ if [ "${ALPAKA_CI_ANALYSIS}" == "ON" ] ;then ./script/travis/install_analysis.sh
 if [ "${ALPAKA_ACC_GPU_CUDA_ENABLE}" == "ON" ] ;then ./script/travis/install_cuda.sh ;fi
 if [ "${CXX}" == "g++" ] ;then ./script/travis/install_gcc.sh ;fi
 if [ "${CXX}" == "clang++" ] ;then source ./script/travis/install_clang.sh ;fi
-if [ "${ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE}" == "ON" ] ;then ./script/travis/install_tbb.sh ;fi
+if [[ ! -v ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE || "${ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE}" == "ON" ]] ;then ./script/travis/install_tbb.sh ;fi
```

Comment:
Why do you install `TBB` if the environment variable `ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE` is not set?

Reply:
Currently we install tbb only if the Flag is ON. However the CMakeLists.txt also uses the backend if the flag was not explicitly given but the backend is available. Now that I use this feature of the CMakeLists, I did not want to add the TBB enable flag to every CI job but have it available by default if this flag is not given to make tue handling equivalrnt to the CMakeLists.txt.
We ma want to differentiate between installation and enabling by using another variable, but this can be done later if it would be required to test such strange combinations.

Reply:
But is this if condition now not always true? If so we can remove the if fully

Reply:
in that case please add a comment over the if condition that it is clear if someone else read the code or if I forget this pr.

Reply:
I have added a comment